README.md

```diff
@@ -1 +1,128 @@
-# node-modules-require
\ No newline at end of file
+# Importing, and Exporting Modules in Node
+
+## Overview
+
+This lesson will cover how to import and export modules in Node.js, what syntax and operators to use and highlight the differences between browser JavaScript and Node when it comes to modules.
+
+## Objectives
+
+1. Describe syntax for working with Node modules
+1. Describe export
+1. Describe require
+1. Describe relationship to CommonJS (contrast with ES6 and browser modules)
+
+## Modules
+
+Modules are distinct chunks of logic typically combined by their functionality (by their purpose). For example, if you are building a date picker, you might want to abstract (separate into a module) your date picker logic into a separate component/package/module, because the chances are very high that you'll need more than one date input field: data of birth, date of expiration on a credit/debit card, date of delivery, etc.
```

Comment:
Let's start the intro by clearly defining a problem:

Let's say you're building an online store and you need to incorporate a date picker to deal with more than one date input field (date of birth, date of expiration on a credit card, date of delivery, etc). if you start writing everything in one single file, you end up with a HUGE file with your date picker logic with your shopping cart and checkout log. 

Just like Ruby stresses separation of concerns, so does JavaScript...

introduce how the browser is so annoying with script tags

THEN introduce and define a module and explain how they make life so much easier